Updated Resource Quota dashboard layout

dtaylor113 opened the pull request:
- Moved 'scopes' to title
- Added 'No Limit/Request' Quota chart placeholders
- Charts now 100% of row
- Removed '--compact' and other unnecessary css selectors

![image](https://user-images.githubusercontent.com/12733153/53355650-7acf6300-38f7-11e9-9c76-193c8ba010bd.png)

JIRA: https://jira.coreos.com/browse/CONSOLE-1257

spadgett commented:
/assign @rhamilto

rhamilto commented on frontend/public/components/_quota.scss:
```diff
@@ -21,41 +18,16 @@
 
-.co-resource-quota-scope__description {
-  font-size: $font-size-small;
+.quota-dashboard-scopes {
```
Add `co-` so the classes are consistent?

dtaylor113 replied:
fixed

rhamilto commented on frontend/public/components/overview/namespace-overview.tsx:
```diff
@@ -46,27 +46,23 @@ const OverviewNamespaceSummary = ({ns}) => <div className="group">
+        {scopes && <QuotaScopesInline className="quota-dashboard-scopes" scopes={scopes} />}
+      </h2>
+    </div>
+    <div className="container-fluid group__body group__graphs">
```
`container-fluid` is not necessary.

dtaylor113 replied:
fixed

rhamilto commented on frontend/public/components/_quota.scss:
```diff
@@ -1,12 +1,9 @@
-.co-resource-quota-chart-row--compact {
-  margin-bottom: 2px;
+  justify-content: space-evenly;
+  margin-left: 40px;
```
I'd be inclined to drop both these margin additions as they result in premature wrapping that's not necessarily desirable.

dtaylor113 replied:
fixed

dtaylor113 commented:
@rhamilto, I believe I have addressed all your review comments.  Yikes!  Didn't realize I created the dev branch in master as opposed to my fork!  -will delete after this PR merges

dtaylor113 commented:
/test console-e2e

rhamilto commented on frontend/public/components/_quota.scss:
```diff
@@ -21,41 +16,16 @@
 
-.co-resource-quota-scope__description {
-  font-size: $font-size-small;
+.co-quota-dashboard-scopes {
```
Sorry, @dtaylor113.  It occurs to me this classname should also include `resource` (e.g., `.co-quota-resource-dashboard-scopes`) and be alphabetized within the file to improve consistency (move rule to line 7).

dtaylor113 replied:
no prob., fixed, thanks :-)

rhamilto commented:
/lgtm

spadgett commented:
/retest

spadgett commented:
/retest

spadgett commented:
/retest

spadgett commented:
/retest

dtaylor113 commented:
/retest

dtaylor113 commented:
/retest

spadgett commented:
/retest

spadgett commented:
/retest

dtaylor113 commented:
/retest

dtaylor113 commented:
/retest

spadgett commented:
olm test flake

/retest

spadgett commented:
/retest

spadgett commented:
install failed

/retest
